.travis.yml

```diff
@@ -9,6 +9,10 @@ before_install:
   - export PATH="$HOME/.meteor:$PATH"
   - pip install --user awscli
   - export PATH=$PATH:$HOME/.local/bin
+  - curl -o $HOME/.local/bin/ecs-cli https://s3.amazonaws.com/amazon-ecs-cli/ecs-cli-linux-amd64-latest && chmod +x $HOME/.local/bin/ecs-cli
+  - aws configure set profile.lmb-dev.aws_access_key_id $AWS_ACCESS_KEY_ID
```

Comment:
Later, when we merge this stuff with staging, we'll need to use the conditional which prevents the script from trying to access the AWS env vars on pull request CI builds.

Reply:
Oh, that's a point. I guess this should all be setup in the deploy stage only.